### PR TITLE
RFE #1561 Being able to use multiple servers at the same time when using cookie auth

### PR DESCRIPTION
**Pull request by madhuracj**
Submitting as a pull requests to get your opinion on changes to common.inc.php. Wonder whether it poses any security threat.

### CODE REVIEW

**Comment by lem9** [comment]
I don't see a security threat; I believe that a threat could happen only if you changed the $token_mismatch logic.

**Comment by madhuracj** [comment]
@lem9 Thanks for the feedback

File: libraries/common.inc.php [hunk 1]
```diff
@@ -872,7 +872,9 @@
 
         if (! $auth_plugin->authCheck()) {
             /* Force generating of new session on login */
-            PMA_secureSession();
+            if ($token_provided) {
+                PMA_secureSession();
+            }
```
Comment (jamesyooda) [inline libraries/common.inc.php]: This code modification does nothing for me, I still cannot connect to multiple servers at the same time, and I still get error: token mismatch

File: libraries/select_server.lib.php [hunk 2]
```diff
@@ -35,7 +35,6 @@ function PMA_selectServer($not_only_options, $omit_fieldset)
                 $GLOBALS['cfg']['DefaultTabServer'], 'server'
             )
             . '" class="disableAjax">';
-        $retval .= PMA_getHiddenFields(array('token' => $_SESSION[' PMA_token ']));
```
Comment (jamesyooda) [inline libraries/select_server.lib.php]: In PMA version 4.6 this line does not exist

File: test/libraries/PMA_select_server_test.php [hunk 3]
```diff
@@ -125,12 +125,6 @@ public function testPMASelectServer()
             $html
         );
 
-        //PMA_URL_getHiddenInputs
-        $this->assertContains(
-            '<input type="hidden" name="token" value="token" />',
-            $html
```
Comment (jamesyooda) [inline test/libraries/PMA_select_server_test.php]: In PMA 4.6 none of these lines exist

**Comment by jamesyooda** [comment]
Your file modifications do not solve the problem, I'm using PMA 4.6. With auth_type in cookie mode PMA is generatiing a different token each time I connect to different sql server, so I will always get a token mismatch, when trying to be connected to multiple databases at the same time. 

Where I work we use lots of different databases, this is a huge problem for us.

**Comment by juokelis** [comment]
As you can see on last comment issue is not solved, with 4.6, now with 4.8 changing server disconnects from previously connected one. Impossible to work with multiple servers in parallel.

**Pull request merged**